lib/block/operation.go

```diff
@@ -23,24 +23,28 @@ type BlockOperation struct {
 	OpHash string `json:"op_hash"`
 	TxHash string `json:"tx_hash"`
 
-	Type   operation.OperationType `json:"type"`
-	Source string                  `json:"source"`
-	Target string                  `json:"target"`
-	Body   []byte                  `json:"body"`
-	Height uint64                  `json:"block_height"`
+	Type    operation.OperationType `json:"type"`
+	Source  string                  `json:"source"`
+	Target  string                  `json:"target"`
+	Body    []byte                  `json:"body"`
+	Height  uint64                  `json:"block_height"`
+	Index   uint64                  `json:"index"`
+	TxIndex uint64                  `json:"tx_index"`
 
 	// bellows will be used only for `Save` time.
 	transaction transaction.Transaction
 	operation   operation.Operation
 	linked      string
+	txIndex     uint64
 	isSaved     bool
+	order       *BlockOrder
 }
 
-func NewBlockOperationKey(opHash, txHash string) string {
-	return fmt.Sprintf("%s-%s", opHash, txHash)
+func NewBlockOperationKey(opHash, txHash string, index uint64) string {
+	return fmt.Sprintf("%s-%s-%d", opHash, txHash, index)
```

Comment:
How about make a hash using those arguments?
@anarcher 

return common.MustMakeObjectHashString([]string{opHash, txHash, index})

Reply:
@kfnagw Really l like it! Make hashing with op,tx,index :-)

Reply:
I will change it! If there is not other opinions :-)